src/Instrumentation/Laravel/src/LaravelInstrumentation.php

```diff
@@ -54,7 +55,7 @@ public static function register(): void
 
                 return [$request];
             },
-            post: static function (Kernel $kernel, array $params, ?Response $response, ?Throwable $exception) {
+            post: static function (Kernel $kernel, array $params, Response|RedirectResponse|null $response, ?Throwable $exception) {
```

Comment:
This looks ok, but don't both these responses share a common interface that might be more appropriate?

Reply:
Good point. It seems that `RedirectResponse` is inherited from `Response`